languages.toml

```diff
@@ -1526,9 +1526,9 @@ source = { git = "https://github.com/victorhqc/tree-sitter-prisma", rev = "17a59
 name = "clojure"
 scope = "source.clojure"
 injection-regex = "(clojure|clj)"
-file-types = ["clj"]
-roots = ["project.clj"]
-comment-token = ";;"
+file-types = ["clj" "cljs" "cljc" "clje" "cljr" "cljx" "edn" "boot"]
+roots = ["project.clj" "build.boot" "deps.edn" "shadow-cljs.edn"]
```

Comment:
for TOML arrays you'll need commas between the elements

Reply:
Ah, thanks for the catch! It's been a while since I manually edited a toml file.